package.json

```diff
@@ -31,7 +31,7 @@
   },
   "dependencies": {
     "@redsift/d3-rs-intl": "~0.1.0",
-    "@redsift/d3-rs-legends": "~0.2.3",
+    "@redsift/d3-rs-legends": "^0.2.5",
```

Comment:
is this meant to be "^"?

Reply:
The change was automatically generated by `npm i @redsift/d3-rs-legends`

Since we are managing the library internally, and do not have plans for it, it really does not really mather.

However, ill just change it to `~` again since all the other dependencies with internal packages follow the same pattern